format_docs.py

```diff
@@ -0,0 +1,30 @@
+import os    
```

Comment:
Not necessary for now since we use `sed` to change some docs. This boilerplate may be useful in the future however.